### PR TITLE
Revert "Pin Facter to < 4"

### PR DESCRIPTION
**Pull request by ekohl**
This reverts commit 011f84bcaab7b07440a763f39ecdf643f70cda76. It temporarily uses the experimental_exit to test whether that fixes it.

cc @IrimieBogdan

### CODE REVIEW

**Comment by ekohl** [comment]
The Puppet 5 failure is irrelevant. On Puppet 6 it looks like it still output the error but doesn't fail.

File: Gemfile [hunk 1]
```diff
@@ -4,9 +4,7 @@
 source 'https://rubygems.org'
 
 gem 'puppet', ENV.key?('PUPPET_VERSION') ? "~> #{ENV['PUPPET_VERSION']}" : '>= 5.5'
-
-# https://github.com/voxpupuli/puppet-selinux/issues/318
-gem 'facter', '< 4'
+gem 'facter', github: 'puppetlabs/facter-ng', branch: 'experimental_exit'
```
Comment (wbclark) [inline Gemfile]: Using the different branch is temporary, yes? As the issue will be resolved soon in the next facter release.

**Comment by ekohl** [comment]
Ready for review now.

**Comment by mmoll** [comment]
@ekohl Travis reporting seems broken, could you open this as another PR?

**Pull request closed without merging**